google_nest_sdm/camera_traits.py

```diff
@@ -278,6 +279,12 @@ async def generate_web_rtc_stream(self, offer_sdp: str) -> WebRtcStream:
         results = response_data[RESULTS]
         obj = WebRtcStream.from_dict(results)
         obj._cmd = self.cmd
+        _LOGGER.debug("Received answer_sdp: %s", obj.answer_sdp)
```

Comment:
This log may be redundant with the existing log of the post response here since i assume the answer is the primary thing in the log:
https://github.com/allenporter/python-google-nest-sdm/blob/e65a9afaad6741129b8209361536014556a70892/google_nest_sdm/auth.py#L124

Reply:
Yes, the response includes the sdp as well, but it is a json string and we need to format it in order to compare to the result sdp which is a multiline string.

However, I am good to remove this line as well. I think we can either
1. remove this line and print the the result obj as json string as well, so it aligns the log format
2. do nothing

What do you think?

Reply:
I don't have a strong opinion either way. If this is significantly more readable then I'll take your word for it just wanted to make sure you know it's already being printed.